bucket/gnutls.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "3.8.2",
+    "description": "GnuTLS is a secure communications library implementing the SSL, TLS and DTLS protocols and technologies around them.",
+    "homepage": "https://www.gnutls.org",
+    "license": "LGPL-2.1-or-later,GPL-3.0-or-later",
```

Comment:
From https://gitlab.com/gnutls/gnutls/-/blob/master/LICENSE :

> The GNU LGPL applies to the main GnuTLS library, while the
included applications as well as gnutls-openssl 
library are under the GNU GPL version 3.